### PR TITLE
[Feature] Location

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # 179452888

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### Page

Screenshot 2021-09-20 at 15-16-48 PesaYetu

### Component Loading

Peek 2021-09-20 15-11
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/ExplorePage/index.js [hunk 1]
```diff
@@ -81,19 +105,24 @@ function ExplorePage({
...
+        zoom={6.25}
+        geometries={geometries}
+        geography={geography}
+        setShouldFetch={setShouldFetch}
+        setGeoCode={setGeoCode}
+        {...props}
+        className={classes.map}
+      />
+      <Location
+        {...locationArgs}
```
Comment (KhadijaMahanga) [inline src/components/ExplorePage/index.js:122]: Bonus: We could use the geography state variable
Screenshot from 2021-09-20 16-10-50
a

**Review by KhadijaMahanga — APPROVED** [review]
🚀 🚀

**Review by KobbyMmo — COMMENTED** [review]
🚀🚀  Looks good @kilemensi, but are we updating the county information on click? It is the same, whichever county is selected. 

Screenshot 2021-09-20 at 1 22 26 PM

**Comment by kilemensi** [comment]
> Looks good @kilemensi, but are we updating the county information on click? It is the same, whichever county is selected.
> 

It's a bonus @KobbyMmo  https://github.com/CodeForAfrica/PesaYetu/pull/96#discussion_r712150797

File: src/components/ExplorePage/index.js (hunk 1 above)
Comment (kilemensi) [reply src/components/ExplorePage/index.js:122]: In a sub-county/district @KhadijaMahanga , will parents be [_county_, **country**] or [**country**, _county_] order?

**Pull request merged**